api/v1/flow/flow.proto

```diff
@@ -156,6 +159,16 @@ message Layer7 {
     }
 }
 
+// Trace contains information about a distributed trace.
+message Trace {
+    TraceContext context = 1;
+}
+
+// TraceContext is metadata about trace spans.
+message TraceContext {
+    string trace_id = 1;
+}
```

Comment:
We get more info from the [Traceparent Header](https://www.w3.org/TR/trace-context/#traceparent-header) but we only need `trace-id` correct?

Reply:
Yes, we only care about the trace ID for now. However, I think we should be forward thinking about our API which is why there is one more level of indirection.
In fact, based on your feedback and @lambdanis, I updated the PR such that now a flow contains a `TraceContext` field which is modeled after the W3C Trace Context specification. The `TraceContext` message contains a `TraceParent` message which itself contains the `trace_id` as a hex string.